java-security-it/src/test/java/com/sap/cloud/security/test/integration/IasIntegrationTest.java

```diff
@@ -0,0 +1,39 @@
+package com.sap.cloud.security.test.integration;
+
+import com.sap.cloud.security.config.OAuth2ServiceConfiguration;
+import com.sap.cloud.security.config.Service;
+import com.sap.cloud.security.config.cf.VcapServicesParser;
+import com.sap.cloud.security.test.SecurityTestRule;
+import com.sap.cloud.security.token.*;
+import com.sap.cloud.security.token.validation.CombiningValidator;
+import com.sap.cloud.security.token.validation.ValidationResult;
+import com.sap.cloud.security.token.validation.validators.JwtValidatorBuilder;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import java.io.IOException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class IasIntegrationTest {
+
+	@ClassRule
+	public static SecurityTestRule rule = SecurityTestRule.getInstance(Service.IAS)
+			.setKeys("/publicKey.txt", "/privateKey.txt");
```

Comment:
which token keys do yu make use of here? as IAS token keys looks slightly different...

Furthermore is there a central place where these mock server endpoints are configured?
See also here:
https://github.wdf.sap.corp/CPSecurity/Knowledge-Base/blob/master/99_Internal/03_ApplicationSecurity/IntegrationTests/README.md#mock-server-oidc-discovery-endpoint